docs/changelog.md

```
@@ -5,6 +5,12 @@ description: The latest updates and changes to CodeRabbit.
 sidebar_position: 9
 ---
 
+## Microsoft Teams Stand Up Reporting
+#### Oct 30, 2024
+
+- Added support for Standup Up scheduled reporting features for Microsoft Teams over webhooks
+
+
```

Markdownlint

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)